src/main/webapp/app/home/generator/jhipster.configuration.model.ts

```diff
@@ -18,33 +18,31 @@
  */
 
 export class JHipsterConfigurationModel {
-    constructor(
-        public applicationType: string,
-        public gitCompany: string,
-        public baseName: string,
-        public packageName: string,
-        public packageFolder: string,
-        public serverPort: Number,
-        public serviceDiscoveryType: any,
-        public authenticationType: string,
-        public uaaBaseName: string,
-        public cacheProvider: string,
-        public enableHibernateCache: boolean,
-        public websocket: any,
-        public databaseType: string,
-        public devDatabaseType: string,
-        public prodDatabaseType: string,
-        public searchEngine: any,
-        public enableSwaggerCodegen: any,
-        public messageBroker: any,
-        public buildTool: string,
-        public useSass: Boolean,
-        public clientPackageManager: string,
-        public testFrameworks: string[],
-        public enableTranslation: Boolean,
-        public nativeLanguage: string,
-        public languages: string[],
-        public clientFramework: string,
-        public jhiPrefix: string
-    ) {}
+    public applicationType = 'monolith';
+    public gitCompany = '';
+    public baseName = 'jhipsterSampleApplication';
+    public packageName = 'com.mycompany.myapp';
+    public packageFolder = 'com/mycompany/myapp';
+    public serverPort = 8080;
+    public serviceDiscoveryType: any = false;
+    public authenticationType = 'jwt';
+    public uaaBaseName = '../uaa';
+    public cacheProvider = 'ehcache';
+    public enableHibernateCache = true;
+    public websocket: any = false;
+    public databaseType = 'sql';
+    public devDatabaseType = 'h2Disk';
+    public prodDatabaseType = 'mysql';
+    public searchEngine: any = false;
+    public enableSwaggerCodegen: any = false;
```

Comment:
For every boolean initialized here, it can be simplified like `public enableSwaggerCodegen = false; `

Reply:
I thought the same thing but in **generator.component.ts** we have:

```
if (this.model.searchEngine) { // <-- here it's a boolean
  this.model.searchEngine = 'elasticsearch'; // <-- and here affected to a string
}
```
So it doesn't compile with `public searchEngine = false;` (same thing for enableSwaggerCodegen, messageBroker and websocket)

Maybe we can change this and use `string` and manage with undefined, null or empty string instead of a false. But I don't know the consequence in the generator on how the variable is handled.

Reply:
Oh yes sorry I didn't notice, `npm run test` and `npm run lint` were both successful.

Same as you, I don't think that's really safe to change the value `false` by something else here, as we don't know if side effects could appear in the generator. Moreover, this weakness is already present in the (JavaScript) generator.

A TypeScript way to resolve this could be to create a type able to manage `string` or `false`:
```ts
type stringOrFalse = string | false;
export class JHipsterConfigurationModel {
    // ...
    public serviceDiscoveryType: stringOrFalse = false;
    // ...
}
```
But I'm not sur that's better than a simple `any`...

If you're ok, I merge your PR with the current code.

Reply:
nice, okay for me. Thanks for the review.

I've just amended the commit to put `public languages = ['en'];` instead of `public languages: string[] = ['en'];` as I added `'en'` in the list recently.